icon/parse

```diff
@@ -92,11 +92,9 @@ fi
 
 COMPUTER=$(hostname)
 
-#if [[ $COMPUTER == *"daint"* ]]
-#then
-#  COMPUTER="daint"
-#  CMAKE_MOD="CMake"
-if [[ ${COMPUTER} == *"kesch"* ]]; then
+if [[ $COMPUTER == *"daint"* ]]; then
+  COMPUTER="daint"
+elif [[ ${COMPUTER} == *"kesch"* ]]; then
```

Comment:
shall we already add Tsa ?

Reply:
Adding Tsa also requires a tsa-specific compile config file. I'd say we add that when needed.